docs/install/Knative-with-Ambassador.md

```diff
@@ -158,5 +147,7 @@ manage and route to serverless applications!
 - Try the
   [Getting Started with App Deployment guide](./getting-started-knative-app/)
   for Knative serving.
-- Take a look at the rest of what
-  [Knative has to offer](https://knative.dev/docs/index.html)
+- Get started with Knative Eventing by walking through one of the
+  [Eventing Samples](../eventing/samples/).
+- [Install Cert-Manager](../serving/installing-cert-manager.md) if you want to use the
```

Comment:
At this point, auto TLS only works with Istio. So I would not put auto TLS under Ambassador installation guide.

Reply:
Removed.